website/content/docs/job-specification/periodic.mdx

```diff
@@ -59,6 +59,9 @@ consistent evaluation when Nomad spans multiple time zones.
   prevents this job from running on the `cron` schedule but prevents force
   launches.
 
+See the [parameterized] documentation for additional considerations when
+using in conjunction with parameterized options.
```

Comment:
```suggestion
Refer to the [parameterized] documentation for how to use parameters with a periodic job.
```

Style guide: use "refer to" instead of "see"; direct language